src/Swarm/TUI/Model.hs

```diff
@@ -282,11 +288,24 @@ replIndexIsAtInput repl = repl ^. replIndex == replLength repl
 -- UI state
 ------------------------------------------------------------
 
-data Modal
+data ModalType
   = HelpModal
   | WinModal
+  | QuitModal
   deriving (Eq, Show)
 
+data Modal = Modal
+  { _modalType :: ModalType
+  , _modalDialog :: Dialog Bool
+  , _modalWidget :: Widget Name
+  }
+
+makeLenses ''Modal
+
+-- | Width cap for modal and error message windows
+maxModalWindowWidth :: Int
```

Comment:
What is the deeper reason to move `maxModalWindowWidth` from `View.hs` to `Model.hs`? It does not seem to interact with the rest of `Model.hs`.

Reply:
Oh, good catch, no reason. I was moving some things around and that got left behind.  I will move it back.